plugins/blocks/browse/block.tpl

```diff
@@ -28,7 +28,7 @@
 				</li>
 			{/if}
 
-			{if $browseCategories}
+			{if $browseCategories->count}
```

Comment:
Good idea! I've updated those variable names and the docs explaining them to be more clear.